docs/explanation/upgrade.rst

```diff
@@ -27,6 +27,12 @@ in a sequential manner, performing upgrades on units within a single zone in par
 *all-in-one* method for upgrades. because its charm is designed to maintain service
 availability throughout the upgrade process.
 
+
+SLURP support
+~~~~~~~~~~~~~
+
+COU aligns with Charmed OpenStack's upgrade strategy. Since Charmed OpenStack does not support SLURP, COU also does not provide support for it.
+
 .. LINKS
 .. _all-in-one: https://docs.openstack.org/charm-guide/latest/admin/upgrades/openstack.html#perform-the-upgrade
-.. _paused-single-unit: https://docs.openstack.org/charm-guide/latest/admin/upgrades/openstack.html#perform-the-upgrade
\ No newline at end of file
+.. _paused-single-unit: https://docs.openstack.org/charm-guide/latest/admin/upgrades/openstack.html#perform-the-upgrade
```

Comment:
What changed on this line?

Reply:
No new line at the end.

Reply:
Hmm usually github ui would show that case. Also, if I show the commit from my local git, it doesn't show a change on this line. Weird. :thinking: